ports/stm/boards/stm32f411ce_Iskra_JS_mini/pins.c

```diff
@@ -0,0 +1,47 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    { MP_ROM_QSTR(MP_QSTR_B12), MP_ROM_PTR(&pin_PB12) },
+	{ MP_ROM_QSTR(MP_QSTR_LED_GRN), MP_ROM_PTR(&pin_PB12) },
```

Comment:
Having a plain `LED` name will make it work with regular blink examples.

```suggestion
    { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_PB12) },
    { MP_ROM_QSTR(MP_QSTR_LED_GREEN), MP_ROM_PTR(&pin_PB12) },
```